docker-compose.yml

```diff
@@ -71,6 +72,8 @@ services:
       - mysql_data:/var/lib/mysql
     ports:
       - '3306:3306'
+    cap_add:
+      - SYS_NICE
```

Comment:
Could you comment or point us to the documentation as to why this is needed?

Reply:
This is not compulsary and I shall be removing them
I am still into this issue